coral/src/app/components/DisabledButtonTooltip.tsx

```diff
@@ -0,0 +1,61 @@
+import { Button, Tooltip } from "@aivenio/aquarium";
+import { useState } from "react";
+import { ResolveIntersectionTypes } from "types/utils";
+import classes from "src/app/components/DisabledButtonTooltip.module.css";
+
+/** DisabledButtonTooltip shows a disabled looking Primary Button
+ * either with role link or button and a tooltip
+ * this workaround is needed since DS tooltip do not work in react
+ * strict mode at the moment. It also adds accessibility, which the
+ * DS Button with tooltip does not provide (since it's disabled and
+ * the tooltip text is not accessible for SR)
+ **/
+type DisabledButtonTooltipProps = ResolveIntersectionTypes<{
+  children: string;
+  tooltip: string;
+  role?: "button" | "link";
+}>;
+function DisabledButtonTooltip({
+  children,
+  ...props
+}: DisabledButtonTooltipProps) {
+  const [tooltipOpen, setTooltipOpen] = useState(false);
+  const toggleTooltip = () => setTooltipOpen(!tooltipOpen);
+  const elementRole = props.role ? props.role : "button";
+
+  return (
+    <Tooltip
+      // aria-hidden is currently not forwarded, but it is added for semantics purposes
+      // while the tooltip has role "tooltip", it's not in the text
+      // near the element, so screen reader won't be able to access the
+      // information correctly. Also the element related to the tooltip
+      // does not build a relation by using aria-describedby.
+      aria-hidden="true"
+      content={props.tooltip}
+      placement="bottom"
+      isOpen={tooltipOpen}
+      data-testid={"test"}
+    >
+      <div className={classes.buttonWrapper}>
+        <Button.Primary
+          role={elementRole}
+          aria-disabled={true}
+          aria-label={`${children}. ${props.tooltip}`}
+          // this event handling is not ideal, since a button in focus will
+          // behave unexpected when mouse and keyboard are used. since it's
+          // an edge case and we (hopefully) can use tooltip soon, it is ok
+          onMouseEnter={toggleTooltip}
+          onMouseLeave={toggleTooltip}
```

Comment:
I remember i had issues when working on the DS Tooltip that `onMouseLeave` would not fire consistently (it was known issue by the internet as well) and had to fallback to tracking `onMouseMove` and when it leaves the element or something like that.

Though if it works this might be fixed already.

Reply:
Oh, I had that too will adding it, but it looked in my case it was because I had a <div> for tracking the mouse that I had it wrapped around a disabled Button.Primary. After that I did not notice a difference anymore! But I'll do a quick google search and see if that's potential still a problem. Thanks!

Reply:
OK I checked and there's no know issue for that general, it seems to happen in some cases (but not 100% clear when) and seems to be dependent on surrounding elements and reactivity (e.g. one explanation was that it could be caused by child elements of the tracking element being added/removed conditionally). 

I've checked the functionality in our UI and I was REALLY fast with the mouse 😅 😅 😅  and it worked all the time.